UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment/settings.job

```diff
@@ -1,3 +1,3 @@
 {
-  "schedule": "0 0 1 * * *"
+    "schedule": "0 0 1 ? * THU *"
```

Comment:
Is this cron schedule correct? There are 7 entries rather than 6, and there's a question mark which I don't recognise as a valid entry.

Reply:
Good catch, thanks! It seems that Azure uses a different cron library than we thought (NCrontab). I have updated the file with the correct expression. This should be 1am every thursday.